src/state/rootReducer.ts

```diff
@@ -29,7 +30,20 @@ export interface ApplicationState {
   notifications: NotificationState;
 }
 
-export const rootReducer = combineReducers({
+const persistConfig = {
+  key: 'root',
+  blacklist: [
+    'wallets',
+    'authenticators',
+    'authentication',
+    'electrumX',
+    'filters',
+    'toastMessages',
+    NOTIFICATIONS_REDUCER_NAME,
```

Comment:
If NOTIFICATIONS_REDUCER_NAME is on the root's blacklist are whitelisted fields kept on persistor after the app reset?

Reply:
yes, because notificationReducer has its own persistor, check the docs for more info